### DISCUSSION TITLE
HyperHDR, MQTT and Home Assistant

### DISCUSSION THREAD

**Discussion by Jejebond** [community/community#122410] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi @xZetsubou

Are you still working on HyperHDR's MQTT and Home assitant as you mentioned in a HyperHDR's bug report ?
If not, is there an alternate solution ?

Thanks !

**Comment by xZetsubou** [comment]
Hi, When I realized that it was impossible to control other instances on HyperHDR using MQTT I stopped working on the integration however I think it was fixed on [v20](https://github.com/awawa-dev/HyperHDR/releases/tag/v20.0.0.0) Not sure if anything changed on HA side so I'll need time to ensure it works as it should.

**Comment by ebndev** [comment]
Hey @Jejebond & @xZetsubou,

Thanks for participating in GitHub Discussions! Glad you were able to connect however I will be closing this discussion as this is a public forum and not a message board.

Thanks for understanding!
